.github/dockerfiles/ov_build/ubuntu_22_04_android/Dockerfile

```diff
@@ -47,10 +47,10 @@ ENV PATH="$SCCACHE_HOME:$PATH"
 
 # Install Android SDK, NDK and Tools
 ENV ANDROID_TOOLS /deps/android_tools
-ENV ANDROID_NDK_HOME /deps/android_tools/ndk-bundle
+ENV ANDROID_NDK_HOME /deps/android_tools/ndk/26.1.10909125
```

Comment:
Why do we need changes in this file?

Reply:
@EgorDuplensky This change is for Android CI from Aleksandr Voron's fix. @alvoron Could you help explain it?